db/seeds.d/102-organizations.rb

```diff
@@ -5,9 +5,9 @@
 #
 
 Organization.all.each do |org|
-  User.current = User.anonymous_admin
+  original_user, ::User.current = User.current, ::User.anonymous_api_admin
   ForemanTasks.sync_task(::Actions::Katello::Organization::Create, org) unless org.library
-  User.current = nil
+  User.current = original_user
```

Comment:
I have no idea, I'm happy to refactor all seeds files here if it's acceptable to combine it in single PR.